api/api/src/main/scala/uk/ac/wellcome/platform/api/requests/Requests.scala

```diff
@@ -28,6 +29,7 @@ case class V1MultipleResultsRequest(
   @QueryParam includes: Option[V1WorksIncludes],
   @QueryParam query: Option[String],
   @QueryParam _index: Option[String],
+  @QueryParam _queryType: Option[String],
```

Comment:
I’d double-check with Jonathan, but I’d be inclined to skip the V1 API. You can replicate the results (roughly) with the filters, and we’re not really doing much work on it these days. At some point we’ll be freezing its code entirely.

Reply:
Yes do.